docs/notebooks/passband-demo.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "Palomar/ZTF.g",  # SVO identifier of the form "{survey}/{camera}.{filter}"
</pre><p>I'd say "{observatory}", not "{survey}". Single camera may be used for multiple surveys, while it is located at a single observatory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tdastro/pull/382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>